prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -0,0 +1,368 @@
+      documentation: Expects Y, N, or U
+
+    # column AY
+    # todo: convert UNK to U for FL
+    - name: standard.employed_in_healthcare
+      csvFields: [{ name: Employment in Health Care }]
+      documentation: Expects Y, N, or U
+
+    # column AZ
+    # nonstandard
+    - name: occupation
+      type: TEXT
+      referenceUrl: https://covid-19-ig.logicalhealth.org/ValueSet-healthcare-occupation-value-set.html
+      csvFields: [{ name: Occupation }]
+      documentation: |
+        FL expects the SNOMED code that maps to one of the values outlined at [https://covid-19-ig.logicalhealth.org/ValueSet-healthcare-occupation-value-set.html](https://covid-19-ig.logicalhealth.org/ValueSet-healthcare-occupation-value-set.html)
+
+    # column BA
+    # todo: convert UNK to U for FL
+    - name: standard.symptomatic_for_disease
+      csvFields: [ { name: Symptomatic } ]
+      documentation: Expects Y, N, or U
+
+    # column BB
+    # non-standard
+    # TODO: implement symptoms value set
+    - name: disease_symptoms
+      type: TEXT
+      csvFields: [{ name: Symptom }]
+      documentation: Expects a list of the symptoms the patient is experiencing as as a set of SNOMED codes
+
+    # column BC
+    - name: standard.illness_onset_date
+      csvFields: [{ name: Date of Symptom Onset, format: "MM/dd/yyyy" }]
+
+    # column BD
+    # todo: convert UNK to U for FL
+    - name: standard.hospitalized
+      csvFields: [{ name: Hospitalized for this condition }]
+      documentation: Expects Y, N, or U
+
+    # column BE
+    # todo: convert UNK to U for FL
+    - name: standard.icu
+      csvFields: [{ name: In ICU }]
+      documentation: Expects Y, N, or U
+
+    # column BF
+    # todo: convert UNK to U for FL
+    - name: standard.resident_congregate_setting
+      csvFields: [{ name: Resides in Congregate Care setting }]
+      documentation: Expects Y, N, or U
+
+    # column BG
+    # nonstandard
+    # todo: create a value set for this based on SNOMED codes
+    - name: specify_congregate_care_setting
+      type: TEXT
+      csvFields: [{ name: Specify Congregate Care Setting }]
+      referenceUrl: https://confluence.hl7.org/pages/viewpage.action?pageId=86967947
+      documentation: |
+        The type of congregate care setting.
+        Based on the value set specified at [https://confluence.hl7.org/pages/viewpage.action?pageId=86967947](https://confluence.hl7.org/pages/viewpage.action?pageId=86967947) item 7a.
+
+    # column BH
+    - name: standard.pregnant
+      csvFields: [ { name: Pregnancy Status } ]
+
+    # column BI
+    # nonstandard
+    - name: student_teacher_other
+      type: TEXT
+      csvFields: [{ name: "Is the patient a student, teacher, or other faculty member" }]
+      documentation: |
+        AOE question for Florida. Expects one of the following values:
+            - Student
+            - Teacher
+            - Other (Faculty Member)
+            - N
+            - U
+
```

Comment:
Could define a `CODE` with AltValues here.

Reply:
I will convert these to CODE and add AltValues